spec/requests/trashbin_api_spec.rb

```diff
@@ -74,73 +73,137 @@
     let(:resource_kind) { trashed_resource.kind }
     let(:resource_id) { trashed_resource.id }
 
-    let(:payload) {{
-      parent: {
-        kind: parent_kind,
-        id: parent_id
-      }
-    }}
+    context 'without payload' do
+      subject { put(url, headers: headers) }
```

Comment:
Would it work to wrap all these contexts with `it_behaves_like 'a PUT request'`?